translation/source/punishments.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <string name="cleanAllGood">Your account is not marked or restricted. You're all good!</string>
+  <string name="engineMarked">Your account is marked for external assistance in games.</string>
+  <string name="engineMarkedInfo">"We define this as using any external assistance to strengthen your knowledge and, or, calculation ability to gain an unfair advantage over your opponent. Some examples would include computer engine assistance, opening books (except for correspondence games), endgame tablebases, and asking another player for help, although these aren’t the only things we would consider cheating."</string>
```

Comment:
Can we link to this page instead of repeating it here? https://lichess.org/page/fair-play